python/cucim/README.md

```diff
@@ -15,15 +15,16 @@
 
 ### Install cuCIM
 
-```
+```bash
 pip install cucim
 
 # Install dependencies for `cucim.skimage` (assuming that CUDA 11.0 is used for CuPy)
-pip install scipy scikit-image cupy-cuda110==9.0.0b3
+pip install scipy scikit-image cupy-cuda110
```

Comment:
Should we update this as well?

https://github.com/rapidsai/cucim/blob/9e346db3a2c9f3a5be931b614f16d31d0bd88cc6/python/cucim/docs/requirements.txt#L21

Reply:
good catch! 
Updated.